src/StatesDumper/Settings.cs

```diff
@@ -26,10 +26,6 @@ internal class Settings
         /// </summary>
         public uint HeightToBegin { get; }
         /// <summary>
-        /// Height to begin real-time syncing and dumping on, consequently, dumping every block into a single files
-        /// </summary>
-        public int HeightToStartRealTimeSyncing { get; }
```

Comment:
Why did you remove this configuration?
This can be useful when debugging or following the chain on real-time